src/components/DataSubmissions/ValidationStatus.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, styled } from "@mui/material";
+import { useMemo } from "react";
+import StyledTooltip from "../StyledFormComponents/StyledTooltip";
+import RedBell from "../../assets/icons/red_bell.svg";
+import GreenBell from "../../assets/icons/green_bell.svg";
+import { FormatDate, capitalizeFirstLetter } from "../../utils";
+import { useSubmissionContext } from "../Contexts/SubmissionContext";
+
+const StyledChip = styled(Box, { shouldForwardProp: (p) => p !== "variant" })<{
+  variant: "green" | "red";
+}>(({ variant }) => ({
+  backgroundColor: variant === "green" ? "#EDF5F3" : "#FCF7F2",
+  border: `1px solid ${variant === "green" ? "#2A836D" : "#F87C46"}`,
+  borderRadius: "8px",
+  color: variant === "green" ? "#165848" : "#903813",
+  fontWeight: 600,
+  fontFamily: "Public Sans",
+  fontSize: "13px",
+  height: "32px",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  userSelect: "none",
+  maxWidth: "229px",
+  padding: "5px 9px",
+}));
+
+const StyledImage = styled("img")({
+  width: "17px",
+  height: "auto",
+  marginRight: "7px",
+});
+
+/**
+ * Provides the implementation for the ValidationControls current validation Status text.
+ *
+ * @returns {React.FC}
+ */
+export const ValidationStatus: React.FC = () => {
+  const { data } = useSubmissionContext();
+  const { validationStarted, validationEnded, validationScope, validationType } =
+    data?.getSubmission || {};
+
+  const hadValidation: boolean = useMemo<boolean>(
+    () => !!validationStarted || !!validationEnded || !!validationScope || !!validationType,
+    [validationStarted, validationEnded, validationScope, validationType]
+  );
+
+  const typeDescription: string = useMemo<string>(() => {
+    if (validationType?.length === 1 && validationType.includes("file")) {
+      return "Data Files";
+    }
+    if (validationType?.length === 1 && validationType.includes("metadata")) {
+      return "Metadata";
+    }
+
+    return "Both";
+  }, [validationType]);
+
+  // No validation has ever been run, hide the component
+  if (!hadValidation) {
+    return null;
+  }
+
+  return (
+    <StyledTooltip
+      title={
```

Comment:
Very optional. This title logic seems pretty long, it can potentially be cleaned up and moved to a useMemo variable.

Reply:
I think the I improved it in 67834ff but I'm open to additional suggestions if you have any. I was never a fan of how the title is built.